Resources/doc/mailchimp.md

```diff
@@ -27,8 +27,13 @@ add the following config to `app/config/config.yml`
 ```yml
 sulu_form:
     mailchimp_api_key: %parameter_recommended_for_mailchimp_api_key%
+    mailchimp_subscribe_status: %parameter_recommended_for_mailchimp_subscribe_status%
 ```
 
+## Subscribe Status
+
+Short explanation ... pending, subscribed.
```

Comment:
Describe here how to configure pending status also the status should not be a parameter as this:

To change the subscribe status from `subscribed` to `pending` you need to configure the following:

```
sulu_form:
    mailchimp_subscribe_status: pending
```